[0m21:36:59.693242 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000208466100A0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002084A9E37C0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002084A9E3580>]}


============================== 21:36:59.697243 | 7d305215-a2fd-47a4-96fd-b90a01a09629 ==============================
[0m21:36:59.697243 [info ] [MainThread]: Running with dbt=1.6.0
[0m21:36:59.697243 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'logs', 'profiles_dir': 'C:\\Users\\viswa\\.dbt', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'static_parser': 'True', 'invocation_command': 'dbt init', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m21:37:13.472670 [debug] [MainThread]: Starter project path: C:\Users\viswa\AppData\Local\Programs\Python\Python39\lib\site-packages\dbt\include\starter_project
[0m21:37:13.490914 [info ] [MainThread]: 
Your new dbt project "order_transform" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m21:37:13.492008 [info ] [MainThread]: Setting up your profile.
[0m21:38:21.854179 [info ] [MainThread]: Profile order_transform written to C:\Users\viswa\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m21:38:21.856181 [debug] [MainThread]: Command `dbt init` succeeded at 21:38:21.856181 after 82.18 seconds
[0m21:38:21.857181 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000208466100A0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002084A9D5F70>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002084A9D5130>]}
[0m21:38:21.857181 [debug] [MainThread]: Flushing usage events
[0m21:39:11.531374 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000023F28E64FA0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000023F2B7757C0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000023F2B775580>]}


============================== 21:39:11.534976 | 47780353-feb5-47ed-8ac5-9cd3fb7e283e ==============================
[0m21:39:11.534976 [info ] [MainThread]: Running with dbt=1.6.0
[0m21:39:11.534976 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'version_check': 'True', 'log_path': 'logs', 'debug': 'False', 'profiles_dir': 'C:\\Users\\viswa\\.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt run', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m21:39:11.536001 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path D:\dbt\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m21:39:11.536984 [debug] [MainThread]: Command `dbt run` failed at 21:39:11.536984 after 0.02 seconds
[0m21:39:11.538319 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000023F28E64FA0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000023F2B766070>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000023F24C9A910>]}
[0m21:39:11.538319 [debug] [MainThread]: Flushing usage events
